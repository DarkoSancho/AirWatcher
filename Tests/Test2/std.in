government official
1
2