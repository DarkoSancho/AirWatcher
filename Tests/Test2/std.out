The sensor Sensor2 is working correctly.